app/src/main/java/org/simple/clinic/mobius/MobiusFoo.kt

```diff
@@ -2,6 +2,11 @@ package org.simple.clinic.mobius
 
 import com.spotify.mobius.First
 import com.spotify.mobius.Init
+import com.spotify.mobius.Next
+import com.spotify.mobius.Next.dispatch
 
 fun <M, F> ((M) -> First<M, F>).toInit(): Init<M, F> =
     Init { this.invoke(it) }
+
+fun <M, F> justEffect(effect: F): Next<M, F> =
```

Comment:
Should this take in a `vararg` parameter?

Reply:
I guess we should do the following,

```
fun <M, F> effects(effect: F, vararg effects: F): Next<M, F>
```

and keep the current one as is, thoughts?

Reply:
Works.

Reply:
Maybe choose the name as `justEffects` to retain consistency?

Reply:
I am looking at the patterns in the following features,
* All Patients In Facility
* Edit Patient
* New Entry
* Onboarding

I have enough samples to extract and reuse one or more functions. Can we make this part of a separate PR (I'm working on it now)?

Reply:
Works.

Reply:
While you're doing this, there's also an unused extension function `First.toInit()` that we can remove.